Detections/OfficeActivity/Office_MailForwarding.yaml

```diff
@@ -20,17 +20,17 @@ relevantTechniques:
   - T1020
 query: |
 
-  let timeframe = 7d
+  let timeframe = 7d;
   OfficeActivity
   | where TimeGenerated >= ago(timeframe)
   | where Operation =~ "Set-Mailbox"
   | where Parameters contains "ForwardingSmtpAddress"
   | extend parsed = parse_json(Parameters)
   | extend parameterName = parsed[1].Name, fwdingDestination = tostring(parsed[1].Value)
   | where isnotempty(fwdingDestination)
-  | summarize TimeGenerated=max(TimeGenerated) by UserId, fwdingDestination
-  | summarize TimeGenerated=max(TimeGenerated), userCount = dcount(UserId), UserId = makelist(UserId) by fwdingDestination
+  | summarize TimeGenerated=max(TimeGenerated) by UserId, ClientIP, fwdingDestination
+  | summarize TimeGenerated=max(TimeGenerated), userCount = dcount(UserId), UserId = makelist(UserId), ClientIP = makeset(ClientIP) by fwdingDestination
```

Comment:
this could be a large set / and don't think entity extraction handles sets anyway?

Reply:
We use mvexpand below to break out into single instances, Tested on more than one workspace with varied data, we should be good here.